server/datastore/psql/database.go

```diff
@@ -0,0 +1,13 @@
+package psql
+
+import (
+	"fmt"
+	"os"
+	"github.com/jackc/pgx"
+)
+
+/* Create a Struct that can connect with the database.*/
+type DataBase struct {
```

Comment:
I actually think this struct should go in the datastore file and be used to make sure the parameters passed into DB.Init() are valid.